aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -967,4 +967,9 @@ std::tuple<Tensor, Tensor> cummin(const Tensor& self, Dimname dim) {
 std::tuple<Tensor&, Tensor&> cummin_out(Tensor& values, Tensor& indices, const Tensor& self, Dimname dim) {
   return at::cummin_out(values, indices, self, dimname_to_position(self, dim));
 }
+
+Tensor dist(const Tensor &self, const Tensor& other, Scalar p){
+  return at::norm(self - other, p);
```

Comment:
Could you do a quick benchmark to make sure there are no performance regressions?
Also the implementation in TH was returning high precision result (double if the input is float. long if the input is int, etc). Does this implementation has the same behavior or does it have lower precision?

Reply:
Benchmark has been added. 
TH or native implementation only support float type, so there is no high precision result of int to long.